dev/how-to/deploy/orchestrated/tidb-in-kubernetes/aws-eks.md

```diff
@@ -233,31 +234,240 @@ By default, the terraform script will create a new VPC. You can use an existing
+  # TiKV replica number
+  tikv_count                    = 3
+  # TiKV instance type
+  tikv_instance_type            = "t2.xlarge"
+  # The storage class used by TiKV, if the TiKV instance type do not have local SSD, you should change it to storage class
+  # TiDB replica number
+  tidb_count                    = 2
+  # TiDB instance type
+  tidb_instance_type            = "t2.xlarge"
+  # Monitor instance type
+  monitor_instance_type         = "t2.xlarge"
+  # The version of tidb-cluster helm chart
+  tidb_cluster_chart_version    = "v1.0.0-rc.1"
+}
+```
+
+> **Note:**
+>
+> The `cluster_name` of each cluster must be unique.
 
-### Customize TiDB parameters
+You can get the DNS name of TiDB service and the Grafana service via `kubectl`. If you want Terraform to print this information, you can add `output` sections in `outputs.tf`:
 
-Currently, there are not many customizable TiDB parameters. And there are two ways to customize the parameters:
+```hcl
+output "example-cluster_tidb-hostname" {
+  value = module.example-cluster.tidb_hostname
+}
 
-* Before deploying the cluster, you can directly modify the `templates/tidb-cluster-values.yaml.tpl` file and then deploy the cluster with customized configs.
-* After the cluster is running, you must run `terraform apply` again every time you make changes to the `templates/tidb-cluster-values.yaml.tpl` file, or the cluster will still be using old configs.
+output "example-cluster_monitor-hostname" {
+  value = module.example-cluster.monitor_hostname
+}
+```
 
 ## Destroy
 
-It may take some while to finish destroying the cluster.
+It may take some time to finish destroying the cluster.
 
 {{< copyable "shell-regular" >}}
 
-```shell
+``` shell
 terraform destroy
 ```
 
 > **Note:**
 >
-> You have to manually delete the EBS volumes in AWS console after running `terraform destroy` if you do not need the data on the volumes anymore.
+> * This will destroy your EKS cluster along with all the TiDB clusters you deployed on it.
+> * If you do not need the data on the volumes anymore, you have to manually delete the EBS volumes in AWS console after running `terraform destroy`.
+
+## Manage multiple Kubernetes clusters
+
+This section describes the best practice to manage multiple Kubernetes clusters, each with one or more TiDB clusters installed.
+
+Under the hood, this Terraform module composes several sub-modules:
+
+- `tidb-operator`, that provisions the Kubernetes control plane for TiDB cluster
+- `tidb-cluster`, that creates the resource pool in the target Kubernetes cluster and deploy the TiDB cluster
+-  A `VPC` module, a `bastion` module and a `key-pair` module that are dedicated to TiDB on AWS
+
+The best practice for managing multiple Kubernetes clusters is creating a new directory for each of your Kubernetes clusters, and combine the above modules via Terraform scripts, so that the Terraform states among clusters do not interfere with each other. Here's an example:
```

Comment:
```suggestion
The best practice for managing multiple Kubernetes clusters is creating a new directory for each of your Kubernetes clusters, and combine the above modules according to your needs via Terraform scripts, so that the Terraform states among clusters do not interfere with each other, and it is convenient to expand. Here's an example:
```